README.md

```diff
@@ -308,17 +308,17 @@ $ git clone https://github.com/erikw/restic-automatic-backup-scheduler.git && cd
 
 Make a quick search-and-replace in the source files:
 ```console
-$ find etc bin -type f -exec sed -i.bak -e 's|$INSTALL_PREFIX||g' {} \; -exec rm {}.bak \;
+$ find . -type f -exec sed -i.bak -e 's|{{ INSTALL_PREFIX }}||g' {} \; -exec rm {}.bak \;
```

Comment:
Maybe don’t like https://github.com/erikw/restic-automatic-backup-scheduler/pull/127/files and add ‘use’ just, as doing ‘.’ Would also modify README.md I would be.

Reply:
Ooops, you're right! I forgot about the README and even Makefile 🤦.

This is the list of files where the placeholder is found, there's also `Library` and `ScheduledTask` in there:

```console
❯ rg '\{\{ INSTALL_PREFIX \}\}' --files-with-matches --hidden | sort
bin/cron_mail
bin/restic_backup.sh
bin/restic_check.sh
bin/resticw
etc/cron.d/restic
etc/restic/default.env.sh
etc/restic/_global.env.sh
.git/COMMIT_EDITMSG
.git/logs/HEAD
Library/LaunchAgents/com.github.erikw.restic-backup.plist
Library/LaunchAgents/com.github.erikw.restic-check.plist
Makefile
README.md
ScheduledTask/install.ps1
usr/lib/systemd/system/nm-unmetered-connection.service
usr/lib/systemd/system/restic-backup@.service
usr/lib/systemd/system/restic-check@.service
usr/lib/systemd/system/status-email-user@.service
```

I post a couple of alternatives I see here (below), you can accept here the one you find more suited if any :) After looking around it I think option (A) may be ok. Notice that this is only documentation for the manual setup, hehe.

**Option A. Explicit exclusion (using grep to find matches)**
(Based on the list above) Exclude hidden dirs, readme and makefile from the search.
```suggestion
$ grep -r -l --exclude-dir '.*' --exclude 'Makefile' --exclude 'README.md' --exclude 'CHANGELOG.md' '{{ INSTALL_PREFIX }}' | xargs sed -i -e 's|{{ INSTALL_PREFIX }}||g'
```

This is the output of the first part (before going into `sed -i`):

```console
❯ grep -r -l --exclude-dir '.*' --exclude 'Makefile' --exclude 'README.md' --exclude 'CHANGELOG.md' '{{ INSTALL_PREFIX }}'                                              
bin/resticw
bin/restic_backup.sh
bin/cron_mail
bin/restic_check.sh
ScheduledTask/install.ps1
Library/LaunchAgents/com.github.erikw.restic-check.plist
Library/LaunchAgents/com.github.erikw.restic-backup.plist
usr/lib/systemd/system/status-email-user@.service
usr/lib/systemd/system/restic-backup@.service
usr/lib/systemd/system/restic-check@.service
usr/lib/systemd/system/nm-unmetered-connection.service
etc/restic/default.env.sh
etc/restic/_global.env.sh
etc/cron.d/restic
```

**Option B. Explicit inclusion**
(I'm understanding that `Library` and `ScheduledTask` need to take part of this)
```suggestion
$ find bin etc usr Library ScheduledTask -type f -exec sed -i.bak -e 's|{{ INSTALL_PREFIX }}||g' {} \; -exec rm {}.bak \;
```

But this would leave `sed` to deal with non-text files in other directories created in the future like images added in the future.

---

I guess a different option would be to create a directory `src/` and put all files in there so they get isolated from README, Makefile, etc —but that would be out of the scope of the PR, hehe.

Reply:
I added co-author info in one commit and force-pushed, but if you commit straight from this button either option-a or b, you can add it too:

```
co-authored-by: giuaig <13609224+giuaig@users.noreply.github.com>
```

Reply:
Thanks for investigating!

Option A is fine, if it works also with BSD Gero that is default in macOS. I’m unable to verify this at the moment (you know why), but if you can verify it works on Mac too , then let’s go :)

Reply:
I could test it and ups! It seems that there were inconsistencies in both `grep` and `sed` 😱, really unfortunate! The grep had solution, but the sed I'm afraid don't have or couldn't come up with it.

Let's go for option-b then?

Reply:
Thank you for checking. Option B applied!